attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>"These attention weights <strong>is</strong> then used" -&gt; "These attention weights <strong>are</strong><span class="ql-cursor">﻿</span> then used"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>